frontend/packages/dev-console/src/utils/create-knative-utils.ts

```diff
@@ -0,0 +1,38 @@
+import { k8sCreate, K8sResourceKind } from '@console/internal/module/k8s';
+import { KsServiceModel } from '../models';
+
+export const createKnativeResource = (
+  name: string,
+  namespace: string,
+  imageStream: any,
+): Promise<K8sResourceKind> => {
+  const knativeDeployResource = {
+    kind: 'Service',
+    apiVersion: 'serving.knative.dev/v1alpha1',
+    metadata: {
+      name,
+      namespace,
+    },
+    spec: {
+      runLatest: {
+        configuration: {
+          revisionTemplate: {
+            spec: {
+              container: {
+                image: `${imageStream.name}:${imageStream.tag}`,
+                resources: {
+                  requests: {
+                    cpu: '100m',
+                    memory: '100Mi',
+                  },
+                },
+              },
+            },
+          },
+        },
+      },
+    },
+  };
+
+  return k8sCreate(KsServiceModel, knativeDeployResource);
```

Comment:
As a util, i think it might be better to leave this create call to whomever is using the util. That way the create logic is separated from persisting to the server.

Reply:
@christianvogt I followed how we did in import utils https://github.com/openshift/console/blob/master/frontend/packages/dev-console/src/components/import/import-submit-utils.ts#L38 , to have consistency. Let me know your thoughts will make changes accordingly

Reply:
Fair enough. If this is strictly to support the form, then ok.